cupy/manipulation/add_remove.py

```diff
@@ -1,5 +1,5 @@
 import cupy
-
+from cupy.core._reduction import ReductionKernel
```

Comment:
Importing non-module symbols is prohibited according to the [coding guideline](https://docs-cupy.chainer.org/en/stable/contribution.html#coding-guidelines).
How about using `cupy.ReductionKernel` simply?